github/org_custom_roles.go

```diff
@@ -0,0 +1,38 @@
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OrginizationCustomRoles represents custom repository roles available in specified organization
+type OrginizationCustomRoles struct {
+	TotalCount  *int           `json:"total_count,omitempty"`
+	CustomRoles []*CustomRoles `json:"custom_roles,omitempty"`
+}
+
+// CustomRoles represents information of custom roles
+type CustomRoles struct {
+	ID   *int64  `json:"id,omitempty"`
+	Name *string `json:"name,omitempty"`
+}
+
+// List Custome Roles in Org
```

Comment:
Idiomatic Go requires that the godoc comments start with the name of the method (and the spelling must match):

```suggestion
// ListCustomRepoRoles lists the custom repository roles available in this organization.
// In order to see custom repository roles in an organization, the authenticated user must be an organization owner.
```